STYLE_API_TEMPLATE.md

```diff
@@ -1,6 +1,6 @@
-# API reference page template
+# API reference style guide
```

Comment:
This page was never meant to be a style guide, rather, a template that any external contributor could use to write their own API documentation. Please change this back.

Reply:
Correct. The original template I provided with sections and descriptions was an actual template for external contributors. 

However, after several rewrites this page contains style guidelines now. 
Hence, it should be titled appropriately as to what it currently provides.